gcp_variant_transforms/third_party/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2017 Google Inc.  All Rights Reserved.
```

Comment:
Please remove for now as discussed.

Reply:
I removed the whole third party library from the history and replaced it with the pypi package.